addons/sale/tests/test_sale_to_invoice.py

```diff
@@ -68,19 +68,40 @@ def setUpClass(cls):
             'default_journal_id': cls.journal_sale.id,
         }
 
+    def check_has_invoices(self, has_invoices):
```

Comment:
```suggestion
    def _check_has_invoices(self, order, has_invoices):
```
Give the order as a param, it'll be more generic and easy to use for future/other tests :)